### DISCUSSION TITLE
AI當大廚?

### DISCUSSION THREAD

**Discussion by zilinng** [1101-GS4719-C/Python-Community-GS4719-C#1250] [General]
我完全沒想過AI對餐飲業也有影響，最近看新聞都說餐飲業缺工的情形愈來愈嚴重，如果AI廚師能夠普及的話，相信能很好的解決這個問題。甚至試想我們可以購買AI機器人廚師放在家裡，想吃什麼料理只需要給它食譜就能馬上吃到熱騰騰的美食，光想都覺得好幸福，如果未來類似產品能在市面上普及的話，一定會大大提升我們的生活品質！

https://news.cts.com.tw/cts/general/202303/202303132153409.html

**Comment by DuxtinTai** [comment]
人工智慧的應用真的越來越多，相信AI主廚肯定會比傳統的煮菜機器人強上百倍
連結: https://www.youtube.com/watch?v=KzzprYtw3cQ&ab_channel=%E6%B5%B7%E8%B3%8A%E7%8E%8B%E5%AD%90

**Comment by curtis1017** [comment]
我不這麼認為，因為我覺得煮菜這個技術要透過熟能生巧，不是透過資料方面就可以煮好菜，雖然可能有密技，但我覺得不一定會煮得很好。

**Comment by CCello** [comment]
我認為由AI當大廚可能還需要一段時間,因為烹飪是一門複雜的技術，需要多方面的技能，包括熟悉食材、掌握烹調方法、控制火候等。AI機器人廚師需要具備這些技能，而且需要能夠根據食譜和客人的口味調整烹調方式和配料，這是一個很大的技術挑戰。

**Comment by KevinCheng42** [comment]
我覺得若要讓AI成為大廚，除了軟體方面的技術外，硬體方面如機械手臂的技術也必須與時俱進，因為實際做出來的料理口味會因為機械手臂無法精細調控翻炒力度或食材比例而無法重現用AI模擬的結果，因此我認為要訓練出一位強大的AI大廚還須待科技的推進~

**Comment by Brownnnn78** [comment]
身為一個大吃貨，如果一道料理每一次炒出來的都是同一個味道真的會減少吃飯的樂趣，(前提是不被雷到)，不過如果這種技術普及下去，每間店的味道感覺都不會變化太大，畢竟不同廚師都有自己不同的步驟及經驗，大數據一下去每間餐廳都一樣味道不太優。
但若是在家裡的話就不一樣了，自己買一台就可以吃到世界各地不同廚師的不同口味，只要機器不把家炸掉的話，我肯定買一台。

**Comment by sjh501** [comment]
目前就算有食譜要能夠透過AI和機器人做出食物還是有很多難題，從食材的處理到後續烹調的工作都需要很多的經驗和技術，需要有各式各樣且大樣的數據，最後還要做出來其中的變數很多，例如溫度和火侯。而且就像前面的人說的，這樣一程不變的味道會變成為了吃而吃，大大減少了體會飲食各種變化的樂趣。

**Comment by ziihui** [comment]
雖然AI廚師的普及有可能解決餐飲業缺工的問題，但我認為這樣會對人類文化產生負面影響。
許多料理都是代表著某個地區的文化和歷史，而透過AI廚師的製作，可能會失去這些背景的特色和價值。此外，料理的製作也需要人類的創造力和想像力，讓每道菜肴都能夠展現出獨特的風格和口味，這是AI無法完全取代的。因此，我們還是需要保留人類廚師的存在，並且珍惜每道菜肴背後的文化價值！🤓
